### PR TITLE
Removed references to `filterjson` being a preview feature

### LINKED ISSUE #3176: Stabilize filterJson [ Prisma 4 ]
Part of the Prisma 4 effort: Docs should not have filterJson marked as preview feature anymore.

Andrew's notes:
We need to change:
* https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#json-filters
* https://www.prisma.io/docs/concepts/components/preview-features/client-preview-features
* https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#enable-advanced-filtering

The SME is Cristian

The PR is ready; Cristian has approved it. We can merge this when Prisma 4 goes live.

### PR DESCRIPTION
**Pull request by andrew-walford-prisma**
Removed references in
* /concepts/components/prisma-client/working-with-fields/working-with-json-fields#enable-advanced-filtering
* /concepts/components/preview-features/client-preview-features
* /reference/api-reference/prisma-client-reference#json-filters

### CODE REVIEW

**Review by cprieto — APPROVED** [review]
Great work!

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx [hunk 1]
```diff
@@ -114,25 +114,6 @@ Advanced `Json` filtering is supported by [PostgreSQL](/concepts/database-connec
 
 </Admonition>
 
-### Enable advanced filtering
-
-You can enable advanced filtering by editing the `previewFeatures` field in the `generator` block in your schema, then re-generating the Prisma Client:
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx:119]: Should this maybe turned into a note when they were in preview? There are quite a few versions between 2.23.0 when this was introduced and now, so quite a few people will be using one of these version and might try to use the filters without the preview feature.

Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx:119]: Fair point - I'll add one.

Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx:119]: I've added this now

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx [hunk 2]
```diff
@@ -104,35 +104,22 @@ const createUser = await prisma.user.create({
 
 See also: [Advanced example: Update a nested JSON key value](#advanced-example-update-a-nested-json-key-value)
 
-## Filtering on a <inlinecode>Json</inlinecode> field
+## Filter on a <inlinecode>Json</inlinecode> field
 
-Advanced `Json` filtering is available from [2.23.0 and later](https://github.com/prisma/prisma/releases/tag/2.23.0). In earlier versions, you can [filter on the exact `Json` field value](#filter-on-exact-field-value).
+From v2.23.0, you can filter rows by the data inside a `Json` type. We call this **advanced `Json`filtering**.
```
Comment (matthewmueller) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]:
```suggestion
From v2.23.0, you can filter rows by the data inside a `Json` type. We call this **advanced `Json` filtering**.
```

**Review by matthewmueller — COMMENTED** [review]
Nice job Andrew! I really like how you detailed what's available in what version.

**Comment by dpetrick** [comment]
Is there a comment somewhere in the docs that this capability isn't available on all databases? Postgres, Cockroach and MySQL have it, but the rest don't.

**Comment by janpio** [comment]
I think this might be what you are looking for @dpetrick: https://deploy-preview-3251--prisma2-docs.netlify.app/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filter-on-a-json-field See the admonition with the yellow/orange line on the left under the list of version availability.

**Pull request merged**